### ISSUE TITLE
Crash in RelativeFormatter

### ISSUE THREAD

**Issue by leeweisern** [malcommac/SwiftDate#720]
Hello! We recently released an update using 6.1.0 and have started getting crashes.

The issue seems to be line 179 in RelativeFormatter.swift

Here's where the crash is occurring in our code.
```
open func relativeTime(_ intDate: Double?) -> String {
	guard let intDate = intDate else { return "" }
	let date = Date(timeIntervalSince1970: intDate)
	
	return date.toRelative(style: RelativeFormatter.defaultStyle(),  locale: Locale(identifier: "en_US"))
}
```
This is the date object printed out on the debugger: 

```
▿ 2019-08-16 16:35:49 +0000
  - timeIntervalSinceReferenceDate : 587666149.0
```
<img width="1058" alt="image" src="https://user-images.githubusercontent.com/14964123/74585022-6a862b80-5013-11ea-8d98-6c8d8c33372e.png">

The error message returned by the debugger: 
```
po localeData.flavours[Flavour.long.rawValue]
"Failed to read data for language id: en"
nil
```

**Comment by mustsait** [comment]
@leeweisern Hi, I tried your code and couldn't be able to reproduce the issue. It seems that It can't find the required **en.json** file. Would you ensure if **en.json** file is exists in "SwiftDate/Sources/Formatters/RelativeFormatter/langs/"  ? Sometimes when pushing to remote branch in git, json files can't be added to index automatically and It can be missed out.

**Comment by leeweisern** [comment]
>@leeweisern Hi, I tried your code and couldn't be able to reproduce the issue. It seems that It can't find the required **en.json** file. Would you ensure if **en.json** file is exists in "SwiftDate/Sources/Formatters/RelativeFormatter/langs/"  ? Sometimes when pushing to remote branch in git, json files can't be added to index automatically and It can be missed out.

@mustsait I figured out the bug. The bundle file is not available if we use SPM to install. I switched to pod and it worked

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by fabioknoedtbcgdv** [comment]
@mustsait I have the same problem. Also using SPM but the `en.json` is present (see attachment). I get the exactly same error. Using Pods, works.

<img width="329" alt="image" src="https://user-images.githubusercontent.com/14126548/82430113-30527d00-9a8d-11ea-88b0-fa7e11465910.png">

**Comment by Sam-Spencer** [comment]
Bump! Having the same issue as @fabioknoedtbcgdv

**Comment by timroesner** [comment]
I'm running into the same issue when using SPM with version 6.1

I know that SPM doesn't properly support bundles, however this crash seems to come from force unwrapping the following line:
```swift
return (.long, localeData.flavours[Flavour.long.rawValue] as! [String: Any])
```

This is what I get from the debugger:
![Screen Shot 2020-05-31 at 1 46 11 PM](https://user-images.githubusercontent.com/13894518/83362317-252d0600-a345-11ea-8efe-d002d9a56fb1.png)

**Comment by jannemecek** [comment]
I've been encountering crashes in production on this line as well. They don't seem to occur for all users so I would guess it's not really related to the bundle files not installing.

**Comment by morgz** [comment]
[Linked issue](https://github.com/malcommac/SwiftDate/issues/704) with fix for SPM using a build phase script.

**Comment by timroesner** [comment]
Anybody interested in taking this on now that SPM has full bundle support: https://developer.apple.com/videos/play/wwdc2020/10169/

**Comment by malcommac** [comment]
Fixed in 6.2.0
